src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/HtmlLinkInfo.java

```diff
@@ -45,27 +45,33 @@ public class HtmlLinkInfo extends LinkInfo {
 
     public enum Kind {
         /**
-         * Link with just the element name as label.
+         * Creates a link with just the element name as label.
```

Comment:
(Minor)
The use of `Creates` makes this seem like a doc comment for a method. Enum constants are state/variables, not methods.